plugins/obs-webrtc/whip-output.cpp

```diff
@@ -133,6 +125,13 @@ void WHIPOutput::ConfigureAudioTrack(std::string media_stream_id,
 void WHIPOutput::ConfigureVideoTrack(std::string media_stream_id,
 				     std::string cname)
 {
+	auto encoder = obs_output_get_video_encoder2(output, 0);
```

Comment:
- I would suggest moving this codeblock down to where the results of these getters are actually used (the codec compares)
- We generally don't like using `auto` for boundaries between c/c++ apis. `OBSEncoder` can be used here.

Reply:
done